docs/source/nlp/text_normalization.rst

```diff
@@ -42,18 +62,36 @@ and ``test.tsv``.
     DATE	2005	two thousand five
     PLAIN	and	<self>
     DATE	2008	two thousand eight
-    PUNCT	.	sil
+    PUNCT	.	<self>
```

Comment:
could you add a note on why `sil` is converted to `self` and at what stage?

Reply:
ok will add in preprocessing stage